hera_pspec/pspecdata.py

```diff
@@ -1851,6 +2103,15 @@ def pspec(self, bls1, bls2, dsets, pols, n_dlys=None, input_data_weight='identit
             given an input visibility noise model, and store the output
             in the UVPSpec object.
 
+        save_q : boolean, optional
+            If True, store the results (delay spectra and covariance matrices) for 
+            the unnormalized bandpowers in the UVPSpec object.
+
+        cov_types : list of strs, optional
+            A list of the models on the input covariance matrix (the covariance matrix
+            for data stored in UVData objects). 
+            Default: ['original'] 
```

Comment:
Let's change this to
```Default = 'original'. Options = ['original', 'time_average', ...]```
where in the `...` you should fill in all the possible options